docs/retrieving-stats.rst

```diff
@@ -6,16 +6,37 @@ You may wish to consume your Locust results via a CSV file. In this case, there
 
 First, when running Locust with the web UI, you can retrieve CSV files under the Download Data tab. 
 
-Secondly, you can run Locust with a flag which will periodically save the CSV file. This is particularly useful
+Secondly, you can run Locust with a flag which will periodically save two CSV files. This is particularly useful
 if you plan on running Locust in an automated way with the ``--no-web`` flag::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -t10m
+    locust -f examples/basic.py --csv=example --no-web -n10 -t10m
 
 or for v0.8 (where the ``-t`` option isn't available)::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -c10
+    locust -f examples/basic.py --csv=example --no-web -n10 -c10
```

Comment:
is `-n` switch working? If not (which I suspect) maybe we should either remove it from the example or reintroduce it to the code.

Reply:
good catch, will update - it worked when I _originally_ wrote this functionality :)

Reply:
@aldenpeterson-wf can you update ^^ ?  then let's get this merged.